src/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+
+ export function useLocalStorage(key, initialValue) {
+  const [value, setStoredValue] = useState(() => {
+      const item= window.localStorage.getItem(key);
+
+      return item ? JSON.parse(item) : initialValue;
+  });
+
+  const setValue = value => {
+    setStoredValue(value);
+    window.localStorage.setItem(key, JSON.stringify(value));
```

Comment:
great job accessing local storage! Pop question: Would saving someone's login information be a good use of local storage? Why or why not?